src/Comparer/StubValueComparer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace Diff\Comparer;
+
+/**
+ * @since 3.0
+ *
+ * @license GPL-2.0+
+ * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ */
+class StubValueComparer implements ValueComparer {
```

Comment:
Not a mistake. Providing test doubles for the interfaces you defined if you are a library can be very useful for consumers. For instance NullLogger in the PSR log package.

Follow up sounds good and might make me understand how that would work.